docs/pages/tutorial/emojimon.mdx

```diff
@@ -0,0 +1,17 @@
+# Introduction
+
+By the end of this tutorial you will learn how to create your own EVM application, a pokémon-inspired onchain game called Emojimon, using [MUD](https://v2.mud.dev/). This will help you understand how MUD works and how it can be used to build any other EVM application or onchain game.
```

Comment:
all the links are absolute, and some point to V2.
Can you make links relative?
eg: `https://mud.dev/client-side#recs-the-reactive-ecs-database` -> `/client-side#recs-the-reactive-ecs-database`